interface/main/messages/messages.php

```diff
@@ -393,30 +393,27 @@ class='form-horizontal'
                                                     generate_form_field(array('data_type' => 1, 'field_id' => 'message_status', 'list_id' => 'message_status', 'empty_title' => 'SKIP', 'order_by' => 'title', 'class' => 'form-control'), $form_message_status); ?>
                                                 </div>
                                                 <div class="col-6 col-md-4">
-                                                    <label for="form_patient">
-                                                        <?php
+                                                    <?php
                                                         if ($task != "addnew" && $result['pid'] != 0) { ?>
-                                                            <a class="patLink" onclick="goPid('<?php echo attr(addslashes($result['pid'])); ?>')"><?php echo xlt('Patient'); ?>:</a>
+                                                            <a class="patLink" onclick="goPid('<?php echo attr(addslashes($result['pid'])); ?>')" title='<?php echo attr(xlt('Click me to Open Patient Dashboard')) ?>'><?php echo xlt('Patient'); ?>:</a><label for="form_patient">&nbsp</label>
```

Comment:
around the new title variable: remove the attr() and change xlt() to xla()
(escaping fun :) )